<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChoisirFichierTravail</class>
 <widget class="QDialog" name="ChoisirFichierTravail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fichier de travail Ã  ouvrir</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="label_fichier">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>1024</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelficprojet">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;nom du fichier&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="label_chemin">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>521</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>1024</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelficprojet_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;chemin&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_parcourir">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_parcourir_fichier">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_copier">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>copie depuis un fichier existant</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="label_ficcopie">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>361</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>2048</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_FACZone">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>90</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>50</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>copie depuis un fichier standard</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChoisirFichierTravail</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChoisirFichierTravail</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
